### PR TITLE
Domains: Don't display cancel button if pending registry

### PR DESCRIPTION
**Pull request by aidvu**
Tell users that at pending_registry status a transfer in is not possible to cancel anymore from our side. Only from the losing registrar side.

Test:
- Start a transfer in
- Hack backend to return `pending_registry` status
- Open the domain in domain management
- Make sure no cancel button
- Check the message saying you cannot cancel the transfer in

### CODE REVIEW

File: client/my-sites/domains/domain-management/edit/transfer.jsx [hunk 1]
```diff
@@ -154,6 +148,31 @@ class Transfer extends React.PureComponent {
...
+			return (
+				<VerticalNav>
+					<VerticalNavItem path={ cancelPurchaseLink( selectedSite.slug, domain.subscriptionId ) }>
+						{ translate( 'Cancel Transfer' ) }
+					</VerticalNavItem>
+				</VerticalNav>
+			);
+		}
+
+		const noCancelMessage = translate(
```
Comment (aidvu) [inline client/my-sites/domains/domain-management/edit/transfer.jsx]: @danhauk this message needs some love.

File: client/my-sites/domains/domain-management/domain-management.jsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ import AddGoogleApps from './add-google-apps';
 import ContactsPrivacy from './contacts-privacy';
 import Dns from './dns';
 import Edit from './edit';
-import TransferIn from './edit/transfer-in.jsx';
```
Comment (aidvu) [inline client/my-sites/domains/domain-management/domain-management.jsx]: Eh, bad... :) But hey, fixed it. 😆

File: client/my-sites/domains/domain-management/edit/transfer.jsx [hunk 3]
```diff
@@ -25,31 +25,49 @@ import { errorNotice, successNotice } from 'state/notices/actions';
...
 		}
 
-		const path = cancelPurchaseLink( this.props.selectedSite.slug, domain.subscriptionId );
+		let noCancelNotice;
+		let cancelNavItem;
+		if ( domain.transferStatus === transferStatus.PENDING_REGISTRY ) {
+			noCancelNotice = (
+				<Notice status={ 'is-info' } showDismiss={ false }>
+					{ translate(
+						"The current state of the transfer doesn't allow cancelling. " +
```
Comment (aidvu) [inline client/my-sites/domains/domain-management/edit/transfer.jsx]: Switched to `<Notice />`. Copy neededn @danhauk

**Comment by danhauk** [comment]
@aidvu I pushed up a copy change.

I'm not super crazy about telling people about canceling up front like this. It should really only show a message if that's their actual intent. On this screen, it might not necessarily be their intent, so showing a message about how to cancel is not relevant until they want to.

However, it's not a blocker for this. Let's get this out and I'll think more about what kind of interaction we could have here.

**Comment by aidvu** [comment]
@danhauk in that case, want to move this message to the `Manage Purchase` screen?

**Pull request merged**